pyproject.toml

```diff
@@ -163,7 +163,7 @@ dependencies = [
   "unidecode>=1.3.8,<1.4",
   "user-agents>=2.0,<2.3",
   "weblate-language-data>=2025.2",
-  "weblate-schemas==2024.2"
+  "weblate_schemas @ git+https://github.com/amCap1712/weblate_schemas.git@categories"
```

Comment:
To allow for testing the PR, until a new release is made including the schema changes.